index.html

```diff
@@ -652,26 +652,26 @@ <h3>
         </h3>
         <p>
           To <dfn data-lt="prompt the user to choose|prompting the user to choose" class=
-          "export">prompt the user to choose</dfn> one of several |options| associated with a
-          |descriptor|, the user agent must perform the following steps. This algorithm returns
-          either {{PermissionState/"denied"}} or one of the options.
+          "export">prompt the user to choose</dfn> one or more of several |options| associated with
+          a |descriptor|, the user agent must perform the following steps. This algorithm returns
+          either {{PermissionState/"denied"}} or the selected option(s).
         </p>
         <ol class="algorithm">
           <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"denied"}}, return
```

Comment:
Then we probably want: 

```suggestion
          <li>Assert: If |restrictions| was passed, |restrictions| is less than or equal to the [=list/size=] of |options|.</li> 
          <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"denied"}}, return
```